addons/crm/models/crm_lead.py

```diff
@@ -903,11 +903,6 @@ def unlink(self):
 
     @api.model
     def _get_view(self, view_id=None, view_type='form', **options):
-        if self._context.get('opportunity_id'):
```

Comment:
Cool, I read the commit message and the whole history is present, thanks for getting into history details ^^ . I was unsure this code was still necessary since a while, glad to see we normally don't have to use it anymore.